src/app/services/question.service.ts

```diff
@@ -135,10 +135,10 @@ export class QuestionService {
     */
     return of(new QuestionDto({
       title: questionTitle,
-      body: 'Lorem Ipsum. I am a lovely wall of text and I have contained within a very hard question. What is the meaning of life?',
+      questionBody: 'Lorem Ipsum. I am a lovely wall of text and I have contained within a very hard question. What is the meaning of life?',
```

Comment:
If we want to see the latex rendering working when this is deployed in the cloud, we will need to change this value as well. That's because right now, the web app does not integrate with the real question service and instead returns the prestashed response.